module/zfs/vdev_label.c

```diff
@@ -511,6 +511,8 @@ vdev_config_generate(spa_t *spa, vdev_t *vd, boolean_t getstats,
 		fnvlist_add_uint64(nv, ZPOOL_CONFIG_ASHIFT, vd->vdev_ashift);
 		fnvlist_add_uint64(nv, ZPOOL_CONFIG_ASIZE,
 		    vd->vdev_asize);
+		fnvlist_add_uint64(nv, ZPOOL_CONFIG_MIN_ALLOC,
+		    vdev_get_min_alloc(vd));
```

Comment:
Do you need a line for `ZPOOL_CONFIG_MAX_ALLOC` here as well?

Reply:
This is for an individual top-level vdev where we know the min and max must be the same.  Since it would be redundant I opted to simply omit it.